### ISSUE TITLE
Bug: Visited links should appear differently from non-visited links

### ISSUE THREAD

**Issue by raphj** [tracim/tracim#4380]
## Description and expectations

Users should  be able to know, across  the whole interface at a first glance whether they already visited a link in this browser.
More specifically, a different style should be applied to visited, active, focused and regular links. 

### Version information

- Tracim version: 3.7
- Web browser: [e.g. Google Chrome, Firefox, Safari]
- Version: [e.g. 22, latest]
- Device / System: [e.g. PC, iPhone 6]

## Screenshots

________________________________________________
